packages/indexer-cli/src/commands/indexer/status.ts

```diff
@@ -23,6 +23,7 @@ module.exports = {
   description: 'Check the status of an indexer',
   run: async (toolbox: GluegunToolbox) => {
     const { print } = toolbox
+    print.warning('We about to get statusy!!')
```

Comment:
Is this log intended?

Reply:
Haha no 😆 .  I must not have pushed the final once-over I did to remove debug logs.